src/components/Hero.tsx

```diff
@@ -64,48 +68,66 @@ const Hero = () => {
         </h1>
       </div>
 
-      <div className="mt-11 px-4 gap-x-2.5 py-2.5 justify-between bg-white shadow-2xl rounded-2xl md:min-w-[26.375rem] flex">
-        <img
-          alt="search icon"
-          src={searchNormal}
-        />
-
-        <input
-          ref={searchBoxRef}
-          className="w-full outline-none text-base text-lightSlate"
-          placeholder="Search repositories"
-          type="text"
-          onChange={e => setValueDebounced(e.target.value)}
-          onFocus={() => setFocus(true)}
-          onBlur={() =>
-            setTimeout(() => {
-              setFocus(false);
-            }, 200)}
-        />
-
-        <img
-          alt="command k"
-          className="pt-1.5"
-          src={cmdKIcon}
-        />
-      </div>
-
-      <div className="mt-2.5 flex w-full justify-center relative">
-        {fetchedData.length > 0 && hasFocus && (
-          <div className="flex md:min-w-96 pb-2 absolute z-50 max-w-96 flex-col bg-white rounded-2.5 shadow-2xl">
-            <div className="bg-gray-100 py-2.5 px-10 md:px-3.5 border-b-gray-100 border-b-0.5 rounded-2.5 rounded-b-none w-full">
-              <p className="text-gray-500 text-sm font-semibold">Repository</p>
-            </div>
-
-            {fetchedData.map(data => (
-              <SearchedRepoCard
-                key={data.full_name}
-                data={data}
-              />
-            ))}
-          </div>
-        )}
-      </div>
+      <Combobox
+        as="div"
+        value={comboBoxSelection}
+        onChange={setComboBoxSelection}
+      >
+        <div className="mt-11 px-4 gap-x-2.5 py-2.5 justify-between bg-white shadow-2xl rounded-2xl md:min-w-[26.375rem] flex">
```

Comment:
```suggestion
        <div className="mt-11 px-4 gap-x-2.5 py-2.5  bg-white shadow-2xl rounded-2xl md:min-w-[26.375rem] flex">
```

Justify-between => justify-content: justify-between . It is good behavior, we don't want to change the base Frame.
![Screenshot 2023-02-21 at 09 25 13](https://user-images.githubusercontent.com/37635083/220276377-eff59b3b-1a7d-4b38-8d79-8dd24844c563.png)

Reply:
Could you please add `flex:1`